css/style.css

```diff
@@ -0,0 +1,772 @@
+/* css reset */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+  vertical-align: baseline;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1em;
+  color: #333;
+  height: 100%;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  background: #fff;
+}
+
+a {
+  color: rgb(18, 18, 18);
+  text-decoration: none;
+  font-size: 1.2em;
+  font-weight: 600;
+}
+
+img {
+  cursor: pointer;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: space-between;
+  flex: 1;
+  align-items: center;
+  margin: 0 30px 0 20px;
+  height: 40px;
+  margin-top: 3px;
+}
+
+.left-container {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.right-container {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.nylogo {
+  width: 250px;
+}
+
+.icon {
+  margin: 10px;
+  display: flex;
+  align-items: center;
+}
+
+.space {
+  font-family: helvetica, arial, sans-serif;
+  letter-spacing: 0.7px;
+  line-height: 19px;
+  font-size: 0.8em;
+  font-weight: 700;
+}
+
+.subscribe a {
+  font-size: 0.875rem;
+  font-weight: 600;
+  opacity: 1;
+  transition: opacity 0.7s ease-in-out 0s;
+  text-decoration: underline;
+}
+
+.Thanks {
+  margin-left: 32%;
+  font-size: 15px;
+  font-family: helvetica, arial, sans-serif;
+  text-align: justify;
+  font-weight: bold;
+}
+
+.subscribe1 {
+  margin-left: 37%;
+  text-align: justify;
+  text-decoration: none;
+  color: #326891;
+  font-size: 15px;
+}
+
+.button {
+  text-align: center;
+  background-color: rgb(18, 18, 18);
+  color: #fff;
+  font-size: 1em;
+  height: 32px;
+  letter-spacing: 0.01em;
+  text-transform: none;
+  border-color: 1px solid #000;
```

Comment:
This is an invalid property. When you are declaring border-color property, only put the color values. If you want to use other border properties, use the short-hand `border` property.